app/models/citizen/access_token.rb

```diff
@@ -0,0 +1,21 @@
+module Citizen
+  class AccessToken < ApplicationRecord
+    EXPIRES_AFTER_IN_DAYS = 8
+
+    self.table_name = "citizen_access_tokens"
+
+    belongs_to :legal_aid_application
+
+    encrypts :token, deterministic: true
```

Comment:
Before, all the "data" was JWT encoded for a `SecureData` record.

Now, we encrypt only the `token`. This is what is used in links in emails to Citizens. It is encrypted deterministically so we can query by it.